service/rtc/stat/stat.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package stat
```

Comment:
I guess you looked at things like https://pkg.go.dev/github.com/montanaflynn/stats? (I thought we talked about that recently)

Reply:
Actually I looked at https://pkg.go.dev/golang.org/x/perf/internal/stats but by then I already implemented everything. If we need something more complex I am happy to add a dependency but for now I am thinking it's fine.